bundles/binding/org.openhab.binding.velux/README.md

```diff
@@ -152,10 +131,6 @@ Rollershutter V_DG_M_S  "Velux DG Rolladen Mitte [%d]"  { velux="thing=bridge;ch
 0,100,V_DG_Shutter_Mitte_100"}
 Rollershutter V_DG_M_W  "Velux DG Window Mitte [%d]"    { velux="thing=bridge;channel=shutter#0,V_DG_Window_Mitte_000,10,V_DG_Window_Mitte_010,
 100,V_DG_Window_Mitte_100"}
-
-//
-// end-of-items/velux.items
-//
 ```
 
 ### Sitemap velux.sitemap
```

Comment:
This line should say just "Sitemap". "velux.sitemap" belongs above the code fence that starts the set of examples.